postgresql.conf.jinja

```diff
@@ -300,6 +300,12 @@ log_min_duration_statement = '1500ms'
 log_line_prefix = '%m [%p] %q%u@%d '
 log_timezone = 'Europe/Berlin'
 
+# Specfies queries to be logged, log destination and log level
+log_statement = '{{ log_statement }}'
+log_directory = '{{ log_directory }}'
+log_filename = '{{ log_filename }}'
+logging_collector = {{ logging_collector }}
```

Comment:
As explained above, those settings should only be set if they are set in the pillar - and leave them alone otherwise. (Contrary to hardcoding the current postgresql default, this also ensures compatibility with upcoming releases).

Therefore, you can set the options with an additional `if` like this:


```jinja
{% if wal_buffers is defined %}
wal_buffers = '{{ wal_buffers }}'
{%- endif %}
```